### PR TITLE
Relax transformative properties constraint on derivation chains to only apply to 'grid' items

### PR DESCRIPTION
**Pull request by leo-barnes**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/227.html" title="Last updated on Sep 11, 2024, 5:59 PM UTC (cf76ce5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/227/cc68618...cf76ce5.html" title="Last updated on Sep 11, 2024, 5:59 PM UTC (cf76ce5)">Diff</a>

### CODE REVIEW

**Comment by leo-barnes** [comment]
It was kind of tricky to come up with good wording. I ended up with this:
```diff
-If transformative properties are used in derivation chains (as defined in [[MIAF]]), they shall only be associated with items that are not referenced by another derived item. For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
+Transformative properties shall not be associated with items in a derivation chain (as defined in [[MIAF]]) that serve as an input to a grid item. For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
```

File: index.bs [hunk 1]
```diff
@@ -358,7 +358,8 @@ NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still
 The following constraints are common to files compliant with this specification:
     - <assert>The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].</assert>
     - <assert>The file shall list <code>'[=avif=]'</code> or <code>'[=avis=]'</code> in the [=compatible_brands=] field of the [=FileTypeBox=].</assert>
-    - <assert>If transformative properties are used in derivation chains (as defined in [[MIAF]]), they shall only be associated with items that are not referenced by another derived item.</assert> For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
+    - <assert>Transformative properties shall not be associated with items in a derivation chain (as defined in [[MIAF]]) that serve as an input to a grid item.</assert> For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
+
```
Comment (y-guyon) [inline index.bs:362]: Extra blank line on purpose?

File: index.bs [hunk 2]
```diff
@@ -358,7 +358,8 @@ NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still
 The following constraints are common to files compliant with this specification:
     - <assert>The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].</assert>
     - <assert>The file shall list <code>'[=avif=]'</code> or <code>'[=avis=]'</code> in the [=compatible_brands=] field of the [=FileTypeBox=].</assert>
-    - <assert>If transformative properties are used in derivation chains (as defined in [[MIAF]]), they shall only be associated with items that are not referenced by another derived item.</assert> For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
+    - <assert>Transformative properties shall not be associated with items in a derivation chain (as defined in [[MIAF]]) that serve as an input to a grid item.</assert> For example, if a file contains a grid item and its referenced coded image items, cropping, mirroring or rotation transformations are only permitted on the grid item itself.
```
[the remark this answers is not in the document]
Comment (leo-barnes) [reply index.bs:361]: Good catch!

[the remark this answers is not in the document]
Comment (leo-barnes) [reply index.bs]: Hmmm.. I see `[[!XXX]]` and `[=XXX=]` both being used. Not completely sure what the difference are.

**Comment by leo-barnes** [comment]
Per meeting: There may be a typo, fix that.

**Pull request merged**